EIPS/eip-609.md

```diff
@@ -22,12 +22,12 @@ This specifies the changes included in the hard fork named Byzantium.
   - Block >= 1,700,000 on Ropsten testnet
 - Included EIPs:
   - [EIP 100](./eip-100.md) (Change difficulty adjustment to target mean block time including uncles)
-  - EIP 140 (REVERT instruction in the Ethereum Virtual Machine)
-  - EIP 196 (Precompiled contracts for addition and scalar multiplication on the elliptic curve alt_bn128)
+  - [EIP 140](./eip-140.md) (REVERT instruction in the Ethereum Virtual Machine)
+  - [EIP 196](./eip-196.md) (Precompiled contracts for addition and scalar multiplication on the elliptic curve alt_bn128)
   - [EIP 197](./eip-197.md) (Precompiled contracts for optimal ate pairing check on the elliptic curve alt_bn128)
   - EIP 198 (Precompiled contract for bigint modular exponentiation)
```

Comment:
This wasn't merged yet?

Reply:
No, it's still waiting for updates by @vbuterin adding the rights and the template still needs some formatting. https://github.com/ethereum/EIPs/pull/198

That link can be updated later once it's merged.